### PR TITLE
Remaining bluetooth samples to use inclusive naming 

### PR DESCRIPTION
**Pull request by rugeGerritsen**
From Bluetooth Core spec v5.3:
Master -> Central
Slave -> Peripheral

### CODE REVIEW

**Review by KAGA164 — APPROVED** [review]

File: samples/bluetooth/llpm/README.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ GATT Latency Service
      - BT_GATT_CHRC_WRITE, BT_GATT_PERM_WRITE
 
 This sample transmits data between two development kits to measure the transmission latency in between.
-One of the devices is connected as a *master* and another is connected as a *slave*.
+One of the devices is connected as a *central* and another is connected as a *peripheral*.
```
Comment (greg-fer) [inline samples/bluetooth/llpm/README.rst:60]: Aren't central and peripheral capitalized in the Bluetooth context?

**Review by greg-fer — CHANGES_REQUESTED** [review]
One doubt. Actually two: I think for this kind of housekeeping change we would need a short entry in the changelog (under Samples > Bluetooth samples > one entry for llpm and one for throughput).

**Comment by rugeGerritsen** [comment]
> One doubt. Actually two: I think for this kind of housekeeping change we would need a short entry in the changelog (under Samples > Bluetooth samples > one entry for llpm and one for throughput).

I thought this is not of interested of to the customers. The application works as before, it is only printing something slightly different

File: samples/bluetooth/llpm/README.rst (hunk 1 above)
Comment (rugeGerritsen) [reply samples/bluetooth/llpm/README.rst:60]: I had a quick look through the rest of the documentation, but could not find a consistent use of it. Therefore I kept the format as is

**Review by greg-fer — APPROVED** [review]

**Pull request merged**